src/Storage/Controllers/MessageboxInstancesController.cs

```diff
@@ -617,7 +617,7 @@ private void AddQueryModelToTelemetry(MessageBoxQueryModel queryModel)
             }
             else
             {
-                List<int> savedList = queryModel.InstanceOwnerPartyIdList;
+                List<int> savedList = queryModel.InstanceOwnerPartyIdList!;
```

Comment:
This could potentially be nullable instead. It's just a temporary variable.
```suggestion
                List<int>? savedList = queryModel.InstanceOwnerPartyIdList;
```